src/service/standard-actions-impl.js

```diff
@@ -267,6 +269,33 @@ export class StandardActions {
       return this.handleHide(invocation);
     }
   }
+
+  /**
+   * Handles "toggleClass" action.
+   * @param {!./action-impl.ActionInvocation} invocation
+   * @return {?Promise}
+   */
+  handleToggleClass(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return null;
+    }
+
+    const target = dev().assertElement(invocation.node);
+    const {args} = invocation;
+    const className = user().assertString(args['class']);
+
+    this.resources_.mutateElement(target, () => {
+      if ({}.hasOwnProperty.call(args, 'opt_force')) {
```

Comment:
`force` instead of `opt_force`. I mentioned `opt_force` to imply it is optional, internally we use `opt` prefix but for the action API, `force` would be better.